code/cutscene/ffmpeg/FFMPEGDecoder.cpp

```diff
@@ -55,48 +58,48 @@ double getFrameRate(AVStream* stream, AVCodecContext* codecCtx) {
 	return fps;
 }
 
-CodecContextParameters getCodecParameters(AVStream* stream) {
+CodecContextParameters getCodecParameters(AVStream *stream) {
```

Comment:
I don't know if it's clang or something else, but this PR has the opening braces of *functions* on the previous line rather than aligned on the following line.